test/test_checks.py

```diff
@@ -1,6 +1,18 @@
 from flake8_quotes import QuoteChecker
 import os
-from unittest import TestCase
+import subprocess
+from unittest import expectedFailure, TestCase
+
+
+FLAKE8_VERSION = int(os.environ.get('FLAKE8_VERSION', 2))
```

Comment:
I don't expect versions to be a number; they are usually strings (e.g. SemVer is X.Y.Z). Please remove the `int` conversion and remove the fallback to 2. I intentionally excluded a fallback so we recieve `None` by default and will use exact matches only (so we don't accidentally pass a version like `4`)